File: 389. Find the Difference/Python/Find the Difference.py
class Solution(object):
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        result_ord = 0
        for item in s:
            result_ord ^= ord(item)
        for item in t:
            result_ord ^= ord(item)
        return chr(result_ord)

File: 389. Find the Difference/Java/Find the Difference.java
public class Solution {
    public char findTheDifference(String s, String t) {
        char result = 0;
        for (char item : s.toCharArray()) {
            result ^= item;
        }
        for (char item : t.toCharArray()) {
            result ^= item;
        }
        return result;
    }
}